### PR TITLE
Add lowercase and uppercase mask support

### PR DESCRIPTION
**Pull request by biel-correa**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/JsDaddy/ngx-mask/blob/develop/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #912

## What is the new behavior?
Now developers can use "L" and "U" masks to implement lowercase and uppercase only.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,6 +114,8 @@ For separator you can add thousandSeparator
 | 000.000.000-99 | 048.457.987-98 |
 | AAAA           | 0F6g           |
 | SSSS           | asDF           |
+| UUUU           | ASDF           |
+| SSSS           | asdf           |
```
Comment (crhistianramirez) [inline README.md:118]: @biel-correa I think you meant to add  `LLLL`  instead of `SSSS` (which already exists)

Comment (biel-correa) [reply README.md:118]: Yes, thanks for warning me

**Comment by biel-correa** [comment]
@crhistianramirez did you find anything else?

**Comment by NepipenkoIgor** [comment]
@biel-correa  Hi . Could you add test for  this issues

**Comment by biel-correa** [comment]
@NepipenkoIgor sure, would you mind showing me an example?

**Comment by NepipenkoIgor** [comment]
@biel-correa Could you check in library  https://github.com/JsDaddy/ngx-mask/tree/develop/projects/ngx-mask-lib/src/test. Also please pay attention that we migrated to angular 13. You need to install deps and make reverse merge fro develop

**Comment by NepipenkoIgor** [comment]
@biel-correa  Hi. Looks like the latest changes broke the whole library . Needed to revert and repeat all from scratch. Please make reverse merge from actual develop

**Comment by NepipenkoIgor** [comment]
@biel-correa  Hi . We add this . Please use mask  >13.1.7

**Pull request closed without merging**